week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +25,17 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+const isNumber = typeof(num);
```

Comment:
Tip: this variable naming as a bit misleading because, as a defacto naming convention, programmers start the name of a variable with `is` if it stores or returns a Boolean value. In your case, it stores a string at this point.

Reply:
yes ,I agree with you

Reply:
it could be this
`const isNumber = typeof(num) === "number"`